build-system/tasks/coverage-map/index.js

```diff
@@ -113,7 +117,13 @@ coverageMap.flags = {
   json:
     '  Customize the name of the JSON output from puppeteer (out.json by default).',
   url:
-    '  Set the URL for puppeteer testing, starting with  "http://localhost:8000..." (http://localhost:8000/examples/everything.amp.html by default).',
+    '  Set the URL for puppeteer testing, starting with  "http://localhost:[port number]..." (http://localhost:[port number]/examples/everything.amp.html by default).',
```

Comment:
```suggestion
    '  Set the URL for puppeteer testing, starting with  "http://localhost[:port_number]..." (http://localhost[:port_number]/examples/everything.amp.html by default).',
```
ULTRA-nit: the colon is only present if the optional port number is included. Also added underscores to prevent odd line-breaks in the middle of `port number`